meshtastic/tests/test_node_runtime_content.py

```
@@ -0,0 +1,1103 @@
+        runtime = _NodeAdminContentRuntime(
+            mock_node_for_admin,
+            cache_store=cache_store,
+            response_runtime=response_runtime,
+        )
+
+        with caplog.at_level(logging.DEBUG):
+            result = runtime._send_content_read_request(
+                begin_read_generation=lambda: 1,
+                is_read_generation_active=lambda _generation: True,
+                retire_read_generation=lambda _generation: None,
+                build_request=lambda msg: None,
```

Ruff (0.15.6)

[warning] Unused lambda argument: `msg`

(ARG005)